infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/LogReplicationClientConfigListener.java

```diff
@@ -123,6 +143,7 @@ public void onNext(CorfuStreamEntries results) {
             }
 
             if (impactedSessions != null) {
+                log.info("Sessions that a forced snapshot sync will be triggered: {}", impactedSessions);
```

Comment:
nit: how about "Sessions {} on which a forced snapshot sync will be triggered" ?